<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QPushButton" name="refreshBtn">
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 170);
border:none 
}
QPushButton:hover
{
background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 190);
border:none 
}
QPushButton:pressed
{
background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 255);
border:none 
}</string>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="YiYanText">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="exitBtn">
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 170);
border:none 
}
QPushButton:hover
{
background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 190);
border:none 
}
QPushButton:pressed
{
background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 255);
border:none 
}</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitBtn</sender>
   <signal>clicked(bool)</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
